layouts/jobs/changes.html

```diff
@@ -35,7 +35,7 @@ <h2>{{ .patch }}</h2>
     {{ partial "components/guide_meta_slider.html" . }}
   </div>
 
-  <div class="py-14 bg-no-repeat bg-cover" style="background-image: url('/theme-assets/jobs/{{ lower .Parent.Title }}_resources_background.png');">
+  <div class="py-14 bg-no-repeat bg-cover" style="background-image: url('/theme-assets/jobs/{{ lower $role }}/{{ lower $role }}_resources_background.png');">
```

Comment:
Isn't this supposed to be something like:

```
'/theme-assets/jobs/{{ lower $role }}/{{ lower $job }}_resources_background.png'
```
???

Reply:
if you look at the asset structure changes, we got the role specific assets inside the role folder now, hence the double role parameter.